db/warehouse/migrate/20230501183045_add_unit_occupancy.rb

```diff
@@ -0,0 +1,21 @@
+class AddUnitOccupancy < ActiveRecord::Migration[6.1]
+  def change
+    remove_column :hmis_units, :inventory_id
+    # Units can have a type
+    add_column :hmis_units, :unit_type_id, :integer, null: true
+    # Units can have a size
+    add_column :hmis_units, :unit_size, :integer, null: true
+
+    # Unit records will be used to represent beds
+    drop_table :hmis_beds
+
+    # Add table for storing who has beed assigned to a given unit over time.
+    # active_ranges table is used to store occupancy period.
+    create_table :hmis_unit_occupancy do |t|
+      t.references :unit, null: false, index: true
+      t.references :enrollment, null: false, index: true
+      t.references :hmis_service, null: true, index: true
+      # t.references :custom_service, null: true, index: true
```

Comment:
Do we need to store date ranges in here?

Reply:
Found this further down.

Reply:
Those are stored in the hmis_active_ranges... see https://github.com/greenriver/hmis-warehouse/pull/3027#discussion_r1185153365